website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -251,10 +251,14 @@ To update:
 
 ## Using VS Code extensions
 
-Visual Studio (VS) Code extensions enhance command line tools by adding extra functionalities. The dbt Cloud CLI is fully compatible with dbt Core, however it doesn't support some dbt Core APIs required by certain tools, for example VS Code extensions. 
+Visual Studio (VS) Code extensions enhance command line tools by adding extra functionalities. The dbt Cloud CLI is fully compatible with dbt Core, however, it doesn't support some dbt Core APIs required by certain tools, for example, VS Code extensions. 
 
-To use these extensions, such as dbt-power-user, with the dbt Cloud CLI, you can install it using Homebrew (along with dbt Core) and create an alias to run the dbt Cloud CLI as `dbt-cloud`. This allows dbt-power-user to continue to invoke dbt Core under the hood, alongside the dbt Cloud CLI.
+You can use extensions like [dbt-power-user](https://www.dbt-power-user.com/) with the dbt Cloud CLI by following these steps:
 
+- Install it using Homebrew along with dbt Core.
```

Comment:
These aren't really "steps" as there are no instructions. Should we show how to install from homebrew and create an alias in this section?

Reply:
the homebrew installation is at the beginning of the page so i don't think we should duplicate instructions due to maintenance but happy to link to it!